prow/Makefile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-HOOK_VERSION   = 0.45
+HOOK_VERSION   = 0.46
```

Comment:
Nothing with this PR in particular, but that seems like a pretty bad idea (though I know someone suggested the same to me)